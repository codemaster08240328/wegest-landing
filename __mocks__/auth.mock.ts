/**
 * Email: testusergmail.com
 * password: testuser123
 */

export const signin = async ({
  email,
  password,
}: {
  email: string
  password: string
}) => {
  return new Promise((resolve) => {
    setTimeout(() => {
      resolve(1)
    }, 200)
  }).then(() => {
    if (email === 'testuser@gmail.com' && password === 'testuser123') {
      return {
        token: 'auth.token',
        user: {
          email: 'testuser@gmail.com',
          firstName: 'James',
          lastName: 'Wo',
          gender: 'Male',
          phone: 'xxxxx',
          role: 'user',
        },
      }
    }

    return 'invalid'
  })
}
