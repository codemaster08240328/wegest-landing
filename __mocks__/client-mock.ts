export const clientUnderProcessing = [
  {
    id: '224FDF9255E5F99900',
    firstName: 'Giulia',
    lastName: 'Malu',
    gender: 'Female',
    frequency: 'not classified',
    frequencyVariation: '=',
    techNotes: [
      {
        date: 'April 11, 2021 - 12:23',
        notes: [
          {
            id: '564',
            type: 'Nail Varnish',
            note: 'CCCCCCCC',
          },
        ],
      },
    ],
    serviceAssigned: [
      {
        id: '4038DF9058',
        name: 'California massage',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DF93',
        name: 'Crease',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB94',
        name: 'Cut',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        operator: true,
        operatorId: 'LUC',
      },
    ],
    serviceHistory: [
      {
        id: '4638DB94',
        name: 'Cut',
        days: 68,
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        days: 68,
      },
      {
        id: '4638DF93',
        name: 'Crease',
        days: 66,
      },
    ],
    productAssigned: [
      {
        id: '423FDB9451E5',
        name: 'bbbbprova',
        operator: true,
        operatorId: 'LUC',
      },
    ],
  },
  {
    id: '224FDD9B56E2F19701',
    firstName: 'Sabina',
    lastName: 'Abdalova',
    gender: 'Female',
    frequency: 'monthly',
    frequencyVariation: '-',
    techNotes: [
      {
        date: 'April 12, 2021 - 09:38',
        notes: [
          {
            id: '7',
            type: 'General',
            note: 'AAAAAAAA',
          },
          {
            id: '508',
            type: 'Cut',
            note: 'BBBBBBBB',
          },
          {
            id: '344',
            type: 'Spikes',
            note: 'HHHHHHH',
          },
          {
            id: '324',
            type: 'Shampoo',
            note: 'SSSSSSS',
          },
        ],
      },
      {
        date: 'April 11, 2021 - 12:23',
        notes: [
          {
            id: '564',
            type: 'Nail Varnish',
            note: 'CCCCCCCC',
          },
        ],
      },
    ],
    serviceAssigned: [
      {
        id: '4038DF9058',
        name: 'California massage',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DF93',
        name: 'Crease',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB94',
        name: 'Cut',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        operator: true,
        operatorId: 'LUC',
      },
    ],
    serviceHistory: [
      {
        id: '4638DB94',
        name: 'Cut',
        days: 68,
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        days: 68,
      },
      {
        id: '4638DF93',
        name: 'Crease',
        days: 66,
      },
    ],
    productAssigned: [
      {
        id: '423FDB9451E5',
        name: 'bbbbprova',
        operator: true,
        operatorId: 'LUC',
      },
    ],
  },
  {
    id: '224FDD9A56E6F49500',
    firstName: 'Bahar',
    lastName: 'Abbaspour',
    gender: 'Female',
    frequency: 'monthly',
    frequencyVariation: '-',
    techNotes: [
      {
        date: 'April 11, 2021 - 12:23',
        notes: [
          {
            id: '564',
            type: 'Nail Varnish',
            note: 'CCCCCCCC',
          },
        ],
      },
    ],
    serviceAssigned: [
      {
        id: '4038DF9058',
        name: 'California massage',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DF93',
        name: 'Crease',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB94',
        name: 'Cut',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        operator: true,
        operatorId: 'LUC',
      },
    ],
    serviceHistory: [
      {
        id: '4638DB94',
        name: 'Cut',
        days: 68,
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        days: 68,
      },
      {
        id: '4638DF93',
        name: 'Crease',
        days: 66,
      },
    ],
    productAssigned: [
      {
        id: '423FDB9451E5',
        name: 'bbbbprova',
        operator: true,
        operatorId: 'LUC',
      },
    ],
  },
  {
    id: '224FDD9759E8F89206',
    firstName: 'Abbate',
    lastName: 'Abballati',
    gender: 'Male',
    frequency: 'fortnightly',
    frequencyVariation: '-',
    techNotes: [
      {
        date: 'April 11, 2021 - 12:23',
        notes: [
          {
            id: '564',
            type: 'Nail Varnish',
            note: 'CCCCCCCC',
          },
        ],
      },
    ],
    serviceAssigned: [
      {
        id: '4038DF9058',
        name: 'California massage',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DF93',
        name: 'Crease',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB94',
        name: 'Cut',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        operator: true,
        operatorId: 'LUC',
      },
    ],
    serviceHistory: [
      {
        id: '4638DB94',
        name: 'Cut',
        days: 68,
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        days: 68,
      },
      {
        id: '4638DF93',
        name: 'Crease',
        days: 66,
      },
    ],
    productAssigned: [
      {
        id: '423FDB9451E5',
        name: 'bbbbprova',
        operator: true,
        operatorId: 'LUC',
      },
    ],
  },
  {
    id: '224FDD9658E4F3970D',
    firstName: 'Mobina',
    lastName: 'Abs',
    gender: 'Female',
    frequency: 'quarterly',
    frequencyVariation: '-',
    techNotes: [
      {
        date: 'April 11, 2021 - 12:23',
        notes: [
          {
            id: '564',
            type: 'Nail Varnish',
            note: 'CCCCCCCC',
          },
        ],
      },
    ],
    serviceAssigned: [
      {
        id: '4038DF9058',
        name: 'California massage',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DF93',
        name: 'Crease',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB94',
        name: 'Cut',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        operator: true,
        operatorId: 'LUC',
      },
    ],
    serviceHistory: [
      {
        id: '4638DB94',
        name: 'Cut',
        days: 68,
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        days: 68,
      },
      {
        id: '4638DF93',
        name: 'Crease',
        days: 66,
      },
    ],
    productAssigned: [
      {
        id: '423FDB9451E5',
        name: 'bbbbprova',
        operator: true,
        operatorId: 'LUC',
      },
    ],
  },
  {
    id: '224FDD9752E1F49904',
    firstName: 'Cristina',
    lastName: 'Abdulla',
    gender: 'Female',
    frequency: 'fortnightly',
    frequencyVariation: '-',
    techNotes: [
      {
        date: 'April 11, 2021 - 12:23',
        notes: [
          {
            id: '564',
            type: 'Nail Varnish',
            note: 'CCCCCCCC',
          },
        ],
      },
    ],
    serviceAssigned: [
      {
        id: '4038DF9058',
        name: 'California massage',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DF93',
        name: 'Crease',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB94',
        name: 'Cut',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        operator: true,
        operatorId: 'LUC',
      },
    ],
    serviceHistory: [
      {
        id: '4638DB94',
        name: 'Cut',
        days: 68,
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        days: 68,
      },
      {
        id: '4638DF93',
        name: 'Crease',
        days: 66,
      },
    ],
    productAssigned: [
      {
        id: '423FDB9451E5',
        name: 'bbbbprova',
        operator: true,
        operatorId: 'LUC',
      },
    ],
  },
  {
    id: '224FDE9255E1F79907',
    firstName: 'Erlina',
    lastName: 'Abaring',
    gender: 'Female',
    frequency: 'weekly',
    frequencyVariation: '=',
    techNotes: [
      {
        date: 'April 11, 2021 - 12:23',
        notes: [
          {
            id: '564',
            type: 'Nail Varnish',
            note: 'CCCCCCCC',
          },
        ],
      },
    ],
    serviceAssigned: [
      {
        id: '4038DF9058',
        name: 'California massage',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DF93',
        name: 'Crease',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB94',
        name: 'Cut',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        operator: true,
        operatorId: 'LUC',
      },
    ],
    serviceHistory: [
      {
        id: '4638DB94',
        name: 'Cut',
        days: 68,
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        days: 68,
      },
      {
        id: '4638DF93',
        name: 'Crease',
        days: 66,
      },
    ],
    productAssigned: [
      {
        id: '423FDB9451E5',
        name: 'bbbbprova',
        operator: true,
        operatorId: 'LUC',
      },
    ],
  },
  {
    id: '224FDE9458E3F59601',
    firstName: 'A',
    lastName: 'A',
    gender: 'Female',
    frequency: 'quarterly',
    frequencyVariation: '-',
    techNotes: [
      {
        date: 'April 11, 2021 - 12:23',
        notes: [
          {
            id: '564',
            type: 'Nail Varnish',
            note: 'CCCCCCCC',
          },
        ],
      },
    ],
    serviceAssigned: [
      {
        id: '4038DF9058',
        name: 'California massage',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DF93',
        name: 'Crease',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB94',
        name: 'Cut',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        operator: true,
        operatorId: 'LUC',
      },
    ],
    serviceHistory: [
      {
        id: '4638DB94',
        name: 'Cut',
        days: 68,
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        days: 68,
      },
      {
        id: '4638DF93',
        name: 'Crease',
        days: 66,
      },
    ],
    productAssigned: [
      {
        id: '423FDB9451E5',
        name: 'bbbbprova',
        operator: true,
        operatorId: 'LUC',
      },
    ],
  },
  {
    id: '224FDB9053E6F090',
    firstName: 'Marco',
    lastName: 'Pernando',
    gender: 'Male',
    frequency: 'quarterly',
    frequencyVariation: '-',
    techNotes: [
      {
        date: 'April 11, 2021 - 12:23',
        notes: [
          {
            id: '564',
            type: 'Nail Varnish',
            note: 'CCCCCCCC',
          },
        ],
      },
    ],
    serviceAssigned: [
      {
        id: '4038DF9058',
        name: 'California massage',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DF93',
        name: 'Crease',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB94',
        name: 'Cut',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        operator: true,
        operatorId: 'LUC',
      },
    ],
    serviceHistory: [
      {
        id: '4638DB94',
        name: 'Cut',
        days: 68,
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        days: 68,
      },
      {
        id: '4638DF93',
        name: 'Crease',
        days: 66,
      },
    ],
    productAssigned: [
      {
        id: '423FDB9451E5',
        name: 'bbbbprova',
        operator: true,
        operatorId: 'LUC',
      },
    ],
  },
]

export const clientInPersonalData = [
  {
    id: '224FDB9053E6F090',
    firstName: 'Marco',
    lastName: 'Pernando',
    gender: 'Male',
  },
  {
    id: '224FDE9458E3F59601',
    firstName: 'A',
    lastName: 'A',
    gender: 'Female',
  },
  {
    id: '224FDE9A55E1F79004',
    firstName: 'BB',
    lastName: 'AA',
    gender: 'Male',
  },
  {
    id: '224FDE9557E1F29407',
    firstName: 'AChi',
    lastName: 'AAA',
    gender: 'Female',
  },
  {
    id: '224FDE9255E1F79907',
    firstName: 'Erlina',
    lastName: 'Abaring',
    gender: 'Female',
  },
  {
    id: '224FDD9759E8F89206',
    firstName: 'Abbate',
    lastName: 'Abballati',
    gender: 'Male',
  },
  {
    id: '224FDE9A55E1F2970C',
    firstName: 'Lorenzo',
    lastName: 'Abbandonato',
    gender: 'Male',
  },
  {
    id: '224FDD9A56E6F49500',
    firstName: 'Bahar',
    lastName: 'Abbaspour',
    gender: 'Female',
  },
  {
    id: '224FDF9352E9F19303',
    firstName: 'GAETANO',
    lastName: 'ABBATE',
    gender: 'Female',
  },
  {
    id: '224FDD9B56E2F19701',
    firstName: 'Sabina',
    lastName: 'Abdalova',
    gender: 'Female',
  },
  {
    id: '224FDD9752E1F49904',
    firstName: 'Cristina',
    lastName: 'Abdulla',
    gender: 'Female',
  },
  {
    id: '224FDD9652E0F1920C',
    firstName: 'Nazanin',
    lastName: 'Abedi',
    gender: 'Female',
  },
  {
    id: '224FDF9352E6F69202',
    firstName: 'Nana',
    lastName: 'Abena',
    gender: 'Female',
  },
  {
    id: '224FDD9B56E6F69904',
    firstName: 'Rakim',
    lastName: 'Abid',
    gender: 'Female',
  },
  {
    id: '224FDD9658E4F3970D',
    firstName: 'Mobina',
    lastName: 'Abs',
    gender: 'Female',
  },
  {
    id: '224FDE9152E5F8940D',
    firstName: 'Raeed',
    lastName: 'Abualrod',
    gender: 'Female',
  },
]

export const TechTypologies = [
  {
    id: '7',
    type: 'General',
    note: '',
  },
  {
    id: '508',
    type: 'Skin',
    note: '',
  },
  {
    id: '510',
    type: 'Spikes',
    note: '',
  },
  {
    id: '513',
    type: 'Shampoo',
    note: '',
  },
  {
    id: '564',
    type: 'Nail Varnish',
    note: '',
  },
  {
    id: '548',
    type: 'Treatments',
    note: '',
  },
]
