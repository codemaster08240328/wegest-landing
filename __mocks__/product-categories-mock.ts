export const categories = [
  {
    title: 'BAIN',
    backgroundColor: 'a7a7a7',
    textColor: 'FFFFFF',
    services: [
      {
        id: '143BDC90561',
        title: 'Bain Aura Botanica',
        qualified: true,
      },
      { id: '1638DB9B1', title: 'Balsamo Muschio anti', qualified: true },
      {
        id: '143BDC90511',
        title: 'Crema Anticellulite Elite',
        qualified: true,
      },
      { id: '1630D593541', title: 'Shampoo Verde', qualified: false },
    ],
  },
  {
    title: 'COLORE',
    backgroundColor: 'a7a7a7',
    textColor: 'FFFFFF',
    services: [
      { id: '243BDC90561', title: 'AApofoefe', qualified: true },
      { id: '2638DB9B1', title: 'Niki', qualified: true },
    ],
  },
  {
    title: 'FORNITORE 1',
    backgroundColor: 'a7a7a7',
    textColor: 'FFFFFF',
    services: [
      { id: '343BDC90561', title: 'Colore 10 minuti', qualified: true },
      { id: '3638DB9B1', title: 'Colore completo', qualified: true },
      { id: '343BDC90511', title: 'Colore Future', qualified: true },
    ],
  },
  {
    title: 'FORNITORE 2',
    backgroundColor: 'a7a7a7',
    textColor: 'FFFFFF',
    services: [
      { id: '443BDC90561', title: 'Colore 10 minuti', qualified: true },
      { id: '4638DB9B1', title: 'Colore completo', qualified: true },
      { id: '443BDC90511', title: 'Colore Future', qualified: true },
      { id: '4630D593541', title: 'colore moda estate', qualified: false },
      { id: '463ED593571', title: 'Colore senza amm.', qualified: false },
      { id: '423FDA9A1', title: 'Colore senza ammonia', qualified: false },
    ],
  },
  {
    title: 'FORNITORE 3',
    backgroundColor: 'a7a7a7',
    textColor: 'FFFFFF',
    services: [
      { id: '543BDC90561', title: 'Colore 10 minuti', qualified: true },
      { id: '543BDD9253E21', title: 'Sunshine', qualified: false },
    ],
  },
  {
    title: 'FORNITORE 4',
    backgroundColor: 'a7a7a7',
    textColor: 'FFFFFF',
    services: [
      { id: '643BDC90561', title: 'Colore 10 minuti', qualified: true },
      { id: '643BDC90511', title: 'Colore Future', qualified: true },
      { id: '6630D593541', title: 'colore moda estate', qualified: false },
      { id: '623AD492541', title: 'elumen', qualified: false },
      { id: '6C38DF96581', title: 'Giallo canarino', qualified: false },
      { id: '6430DE9A591', title: 'Manicure 5', qualified: false },
      { id: '6738DA95591', title: 'prova', qualified: false },
      { id: '643BDD9253E21', title: 'Sunshine', qualified: false },
    ],
  },
  {
    title: 'GAMAX',
    backgroundColor: 'a7a7a7',
    textColor: 'FFFFFF',
    services: [
      { id: '743BDC90561', title: 'Colore 10 minuti', qualified: true },
      { id: '7638DB9B1', title: 'Colore completo', qualified: true },
      { id: '763ED593571', title: 'Colore senza amm.', qualified: false },
      { id: '723FDA9A1', title: 'Colore senza ammonia', qualified: false },
      { id: '7D3FD4901', title: 'Deco', qualified: false },
      { id: '7738DA95591', title: 'prova', qualified: false },
      { id: '743BDD9253E21', title: 'Sunshine', qualified: false },
    ],
  },
  {
    title: 'KERASTASE',
    backgroundColor: 'a7a7a7',
    textColor: 'FFFFFF',
    services: [
      { id: '843BDC90511', title: 'Colore Future', qualified: true },
      { id: '8630D593541', title: 'colore moda estate', qualified: false },
      { id: '863ED593571', title: 'Colore senza amm.', qualified: false },
      { id: '823FDA9A1', title: 'Colore senza ammonia', qualified: false },
      { id: '8D3FD4901', title: 'Deco', qualified: false },
      { id: '8738DA95591', title: 'prova', qualified: false },
      { id: '843BDD9253E21', title: 'Sunshine', qualified: false },
    ],
  },
  {
    title: 'LOREAL',
    backgroundColor: 'a7a7a7',
    textColor: 'FFFFFF',
    services: [
      { id: '943BDC90561', title: 'Colore 10 minuti', qualified: true },
      { id: '9638DB9B1', title: 'Colore completo', qualified: true },
      { id: '943BDC90511', title: 'Colore Future', qualified: true },
      { id: '9738DA95591', title: 'prova', qualified: false },
      { id: '943BDD9253E21', title: 'Sunshine', qualified: false },
    ],
  },
  {
    title: 'WELLA',
    backgroundColor: 'a7a7a7',
    textColor: 'FFFFFF',
    services: [
      { id: '1043BDC90561', title: 'Colore 10 minuti', qualified: true },
      { id: '10638DB9B1', title: 'Colore completo', qualified: true },
      { id: '1023FDA9A1', title: 'Colore senza ammonia', qualified: false },
      { id: '10D3FD4901', title: 'Deco', qualified: false },
      { id: '1023AD492541', title: 'elumen', qualified: false },
      { id: '10D38DE95591', title: 'Prova', qualified: false },
      { id: '10738DA95591', title: 'prova', qualified: false },
      { id: '1043BDD9253E21', title: 'Sunshine', qualified: false },
    ],
  },
]
