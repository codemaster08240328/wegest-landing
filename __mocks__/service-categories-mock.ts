export const categories = [
  {
    title: 'Colore2',
    backgroundColor: '1dc499',
    textColor: 'FFFFFF',
    services: [
      { id: '143BDC9056', title: 'Colore 10 minuti', qualified: true },
      { id: '1638DB9B', title: 'Colore completo', qualified: true },
      { id: '143BDC9051', title: 'Colore Future', qualified: true },
      { id: '1630D59354', title: 'colore moda estate', qualified: false },
      { id: '163ED59357', title: 'Colore senza amm.', qualified: false },
      { id: '123FDA9A', title: 'Colore senza ammonia', qualified: false },
      { id: '1D3FD490', title: 'Deco', qualified: false },
      { id: '123AD49254', title: 'elumen', qualified: false },
      { id: '1C38DF9658', title: 'Giallo canarino', qualified: false },
      { id: '1430DE9A59', title: 'Manicure 5', qualified: false },
      { id: '1D38DE9559', title: 'Prova', qualified: false },
      { id: '1738DA9559', title: 'prova', qualified: false },
      { id: '143BDD9253E2', title: 'Sunshine', qualified: false },
    ],
  },
  {
    title: 'Body',
    backgroundColor: '2080e9',
    textColor: 'FFFFFF',
    services: [
      { id: '243BDC9056', title: 'Colore 10 minuti', qualified: true },
      { id: '2638DB9B', title: 'Colore completo', qualified: true },
      { id: '2D38DE9559', title: 'Prova', qualified: false },
      { id: '2738DA9559', title: 'prova', qualified: false },
      { id: '243BDD9253E2', title: 'Sunshine', qualified: false },
    ],
  },
  {
    title: 'Aesthetics',
    backgroundColor: '1a7ea0',
    textColor: 'FFFFFF',
    services: [
      { id: '343BDC9056', title: 'Colore 10 minuti', qualified: true },
      { id: '3638DB9B', title: 'Colore completo', qualified: true },
      { id: '343BDC9051', title: 'Colore Future', qualified: true },
    ],
  },
  {
    title: 'Meches',
    backgroundColor: 'ffb300',
    textColor: 'FFFFFF',
    services: [
      { id: '443BDC9056', title: 'Colore 10 minuti', qualified: true },
      { id: '4638DB9B', title: 'Colore completo', qualified: true },
      { id: '443BDC9051', title: 'Colore Future', qualified: true },
      { id: '4630D59354', title: 'colore moda estate', qualified: false },
      { id: '463ED59357', title: 'Colore senza amm.', qualified: false },
      { id: '423FDA9A', title: 'Colore senza ammonia', qualified: false },
    ],
  },
  {
    title: 'Fold',
    backgroundColor: '607d8b',
    textColor: 'FFFFFF',
    services: [
      { id: '543BDC9056', title: 'Colore 10 minuti', qualified: true },
      { id: '543BDD9253E2', title: 'Sunshine', qualified: false },
    ],
  },
  {
    title: 'Solarium',
    backgroundColor: 'ffe903',
    textColor: 'FFFFFF',
    services: [
      { id: '643BDC9056', title: 'Colore 10 minuti', qualified: true },
      { id: '643BDC9051', title: 'Colore Future', qualified: true },
      { id: '6630D59354', title: 'colore moda estate', qualified: false },
      { id: '623AD49254', title: 'elumen', qualified: false },
      { id: '6C38DF9658', title: 'Giallo canarino', qualified: false },
      { id: '6430DE9A59', title: 'Manicure 5', qualified: false },
      { id: '6738DA9559', title: 'prova', qualified: false },
      { id: '643BDD9253E2', title: 'Sunshine', qualified: false },
    ],
  },
  {
    title: 'Cut',
    backgroundColor: '64b5f6',
    textColor: 'FFFFFF',
    services: [
      { id: '743BDC9056', title: 'Colore 10 minuti', qualified: true },
      { id: '7638DB9B', title: 'Colore completo', qualified: true },
      { id: '763ED59357', title: 'Colore senza amm.', qualified: false },
      { id: '723FDA9A', title: 'Colore senza ammonia', qualified: false },
      { id: '7D3FD490', title: 'Deco', qualified: false },
      { id: '7738DA9559', title: 'prova', qualified: false },
      { id: '743BDD9253E2', title: 'Sunshine', qualified: false },
    ],
  },
  {
    title: 'Treatments',
    backgroundColor: '2d89ef',
    textColor: 'FFFFFF',
    services: [
      { id: '843BDC9051', title: 'Colore Future', qualified: true },
      { id: '8630D59354', title: 'colore moda estate', qualified: false },
      { id: '863ED59357', title: 'Colore senza amm.', qualified: false },
      { id: '823FDA9A', title: 'Colore senza ammonia', qualified: false },
      { id: '8D3FD490', title: 'Deco', qualified: false },
      { id: '8738DA9559', title: 'prova', qualified: false },
      { id: '843BDD9253E2', title: 'Sunshine', qualified: false },
    ],
  },
  {
    title: 'Man',
    backgroundColor: 'f88b17',
    textColor: 'FFFFFF',
    services: [
      { id: '943BDC9056', title: 'Colore 10 minuti', qualified: true },
      { id: '9638DB9B', title: 'Colore completo', qualified: true },
      { id: '943BDC9051', title: 'Colore Future', qualified: true },
      { id: '9738DA9559', title: 'prova', qualified: false },
      { id: '943BDD9253E2', title: 'Sunshine', qualified: false },
    ],
  },
  {
    title: 'Various',
    backgroundColor: 'd6006b',
    textColor: 'FFFFFF',
    services: [
      { id: '1043BDC9056', title: 'Colore 10 minuti', qualified: true },
      { id: '10638DB9B', title: 'Colore completo', qualified: true },
      { id: '1023FDA9A', title: 'Colore senza ammonia', qualified: false },
      { id: '10D3FD490', title: 'Deco', qualified: false },
      { id: '1023AD49254', title: 'elumen', qualified: false },
      { id: '10D38DE9559', title: 'Prova', qualified: false },
      { id: '10738DA9559', title: 'prova', qualified: false },
      { id: '1043BDD9253E2', title: 'Sunshine', qualified: false },
    ],
  },
]
