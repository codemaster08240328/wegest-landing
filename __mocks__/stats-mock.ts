import { IResume } from '@/shared/@types/models/stastics'

export const resume: Record<string, IResume> = {
  month: {
    revenue: 4675.64,
    avgReceiptValue: 203.29,
    fees: 3878.77,
    serviceFees: 3806.25,
    productFees: 72.52,
    attendance: 23,
    newCustomers: 1,
    handledCustomer: 11,
    expenses: 1105.32,
    netIncome: 3570.32,
  },

  today: {
    revenue: 0,
    avgReceiptValue: 0,
    fees: 0,
    serviceFees: 0,
    productFees: 0,
    attendance: 0,
    newCustomers: 0,
    handledCustomer: 0,
    expenses: 0,
    netIncome: 0,
  },

  yesterday: {
    revenue: 166,
    avgReceiptValue: 166,
    fees: 0,
    serviceFees: 0,
    productFees: 0,
    attendance: 1,
    newCustomers: 0,
    handledCustomer: 1,
    expenses: 0,
    netIncome: 166,
  },
}
