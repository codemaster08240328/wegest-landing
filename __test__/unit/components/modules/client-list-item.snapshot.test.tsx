import renderer from 'react-test-renderer'
import ClientListItem from '@/components/modules/client-list-item'
import { clientUnderProcessing } from '@/__mocks__/client-mock'

describe('<ClientListItem> snapshot', () => {
  it('renders correclty with props', () => {
    expect(
      renderer
        .create(<ClientListItem client={clientUnderProcessing[2]} />)
        .toJSON(),
    ).toMatchSnapshot()
  })
})
