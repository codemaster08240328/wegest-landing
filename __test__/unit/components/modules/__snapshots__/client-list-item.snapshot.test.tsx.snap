// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ClientListItem> snapshot renders correclty with props 1`] = `
<div
  className="MuiBox-root MuiBox-root-4 makeStyles-root-1 "
>
  <div
    className="MuiBox-root MuiBox-root-5 makeStyles-avatar-2"
  >
    <div
      bgcolor="#f06292"
      className="MuiAvatar-root Component-root-6 Component-root-8 MuiAvatar-circle MuiAvatar-colorDefault Component-colorDefault-7 Component-colorDefault-9"
      data-testid="el-avatar"
      size={50}
    >
      BA
    </div>
    Bahar Abbaspour
  </div>
  <button
    className="MuiButtonBase-root MuiIconButton-root Component-root-10 Component-root-11 makeStyles-button-3"
    data-testid="el-icon-button"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiIconButton-label"
    >
      <span
        aria-hidden={true}
        className="material-icons MuiIcon-root"
        data-testid="el-icon"
      >
        <span
          aria-hidden={true}
          className="material-icons MuiIcon-root material-icons-outlined"
          data-testid="el-icon"
        >
          info
        </span>
      </span>
    </span>
  </button>
</div>
`;
